dts/bindings/adc/nxp,kinetis-adc16.yaml

```diff
@@ -20,8 +20,38 @@ properties:
     interrupts:
       required: true
 
+    nxp,dma-channel:
+      type: int
+      required: false
+      description: dma channel to use, for periodic trigger shall be 0-3
+
     "#io-channel-cells":
       const: 1
 
+    nxp,clk-source:
+      type: int
+      required: false
+      description: use alternate clock reference source
+
+    nxp,long-sample:
+      type: int
+      required: false
+      description: enable long sample mode
+
+    nxp,continuous-convert:
+      type: boolean
+      required: false
+      description: If use continuous convert
+
+    nxp,high-speed:
+      type: boolean
+      required: false
+      description: If use high speed
+
+    nxp,hw-trigger-src:
```

Comment:
Please remove the nxp prefix, similar to https://github.com/zephyrproject-rtos/zephyr/pull/30515#pullrequestreview-573663237

Reply:
ok